src/javascripts/components/analytics.js

```diff
@@ -0,0 +1,17 @@
+export default function loadAnalytics () {
+  // Load gtm script
```

Comment:
Could you add to the commit message where the GTM script came from, for future reference?

Reply:
Added here https://github.com/alphagov/govuk-design-system/pull/1704/files#diff-7c543a5ec63db981b754c52be4431d355092eef62e71c1c39849a82d51b37062R4